package.json

```diff
@@ -76,6 +76,7 @@
   },
   "packageManager": "yarn@3.5.0",
   "dependencies": {
+    "@amplitude/experiment-js-client": "^1.11.0",
```

Comment:
If anything this belongs in `base-ui`

Reply:
roger that, removed from all `package.json` files except for `libs`